src/ParcelRegistry/Legacy/CommandHandlerModules.cs

```diff
@@ -11,6 +11,9 @@ public static void Register(ContainerBuilder containerBuilder)
                 .RegisterType<ParcelProvenanceFactory>()
                 .SingleInstance();
 
+            containerBuilder.RegisterType<FixGrar1475ProvenanceFactory>().AsSelf();
+            containerBuilder.RegisterType<FixGrar1637ProvenanceFactory>().AsSelf();
```

Comment:
Added them here based upon https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry/Legacy/CommandHandlerModules.cs#L18-L20

Don't know how they were previously resolved (and if there's a conflict now)?

Reply:
Just saw that ParcelCommandHandlerModule is also created manually in import. Didn't notice that before.